includes/core/functions.php

```diff
@@ -1601,6 +1601,64 @@ function wct_is_signup_allowed_for_current_blog() {
 	return apply_filters( 'wct_is_signup_allowed_for_current_blog', wct_allow_signups() );
 }
 
+/**
+ * Updates last_status post meta before saving talk.
+ *
+ * @since  1.2.0
+ */
+function wct_talks_update_last_status( $data, $postarr ) { 
+	// Checks if we are in 'talks' CPT and if not, abort
+	if( $data['post_type'] !== 'talks' ) :
+		return $data;
+	endif;
+	// Gets current talk status
+	$current_post_status = get_post_status( $postarr['ID'] );
+	// Updates last talk status meta
+	update_post_meta( $postarr['ID'], '_wct_talks_last_status', $current_post_status );
+	return $data;
+}
+
+/**
+ * Run a scheduled task to send emails for talks status updated the last 24h.
+ *
+ * @since  1.2.0
+ */
+function wct_talks_updated_talks_send_emails() {
+	// Gets all talks modified the last 24h
+	$args = array(
+		'post_type'			=> 'talks',
+		'post_status'		=> 'any',
+		'posts_per_page' 	=> -1,
+		'date_query' => array(
+			array(
+				'column' => 'post_modified_gmt',
+				'after'  => '1 day ago',
+			),
+		),
+	);
+	$query_talks = new WP_Query( $args );
```

Comment:
Ok,I wasn't sure about using your functions and I don't had a deep look on these, so I trust you in any case :)

Reply:
Yes i still have a doubt on one thing. Have you tested being logged in as an admin ? Because the WP Cron might fire when a user is not logged in, meaning the read_private_talks cap might be missing. So we should probably temporarly add a filter to map caps before using `wct_talks_get_talks()` and remove this filter right after. Eg:

```php
add_filter( 'map_meta_caps', 'a_function_that_returns_array_exists' );

$q = wct_talks_get_talks( $args );

remove_filter( 'map_meta_caps', 'a_function_that_returns_array_exists' );
```